setup.py

```diff
@@ -8,9 +8,10 @@
     description="Parallel Processing Coordinator. Splits dataset processing to run parallel cluster jobs and aggregates outputs",
     author_email="dataanalysis@diamond.ac.uk",
     packages=find_packages(),
-    install_requires=["h5py", "numpy", "PyYAML", "pydantic"],
+    install_requires=["h5py", "pydantic", "pyyaml", "requests"],
```

Comment:
Is numpy not needed?

Reply:
It is but h5py depends on it so it will always be pulled in by that